src/software/simulated_tests/validation_functions/ball_velocity_validation.cpp

```diff
@@ -0,0 +1,17 @@
+#include "software/simulated_tests/validation_functions/ball_velocity_validation.h"
+
+void ballExceedsVelocity(std::shared_ptr<World> world_ptr,
+                         ValidationCoroutine::push_type& yield)
+{
+    auto ball_exceeded_velocity = [](std::shared_ptr<World> world_ptr) {
+        Vector ball_velocity = world_ptr->ball().velocity();
+
+        // The ball cannot exceed a speed of 6.5 m/s as regulated by the SSL
+        return ball_velocity.length() < 6.5;
```

Comment:
The function is called `ballExceedsVelocity`, but if the `ball_velocity.length()` exceeds 6.5, it returns false. This seems counter-intuitive to the function name. Consider flipping it around.